rusk/tests/common/wallet.rs

```diff
@@ -4,137 +4,47 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use std::collections::HashMap;
-use std::fmt::Debug;
-use std::sync::{Arc, RwLock};
-
-use crate::common::block::Block as BlockAwait;
-
-use dusk_bytes::Serializable;
-use execution_core::{
-    signatures::bls::PublicKey as BlsPublicKey,
-    stake::StakeData,
-    transfer::{
-        moonlight::AccountData,
-        phoenix::{Note, NoteOpening, ViewKey},
-    },
-    BlsScalar,
-};
-use futures::StreamExt;
```

Comment:
Can we finally remove the dependency and the rusk feature too?

Reply:
I think we can!